### PR TITLE
#2432_Added AbstractBLogic prevents creation of proxy class

### PR DESCRIPTION
**Pull request by neetapimpalnerkar**
Please review #2432.

### CODE REVIEW

File: source_en/ImplementationAtEachLayer/DomainLayer.rst [hunk 1]
```diff
@@ -1237,7 +1266,7 @@ Below are the points to be taken care of while creating Service class.
 
     #. If interface is there, When using AOP, Dynamic proxies functionality of standard JDK is used.
        In case of no interface, CGLIB included in Spring Framework is used. In case of CGLIB there are certain restrictions like "Advice cannot be applied on final methods" etc.
-       Refer to \ `Spring Reference Document -Aspect Oriented Programming with Spring(Proxying mechanisms)- <http://docs.spring.io/spring/docs/4.3.5.RELEASE/spring-framework-reference/html/aop.html#aop-proxying>`_\ for details.
+       Refer to \ `Spring Reference Document -Aspect Oriented Programming with Spring(Proxying mechanisms)- <http://docs.spring.io/spring/docs/4.2.7.RELEASE/spring-framework-reference/html/aop.html#aop-proxying>`_\ for details.
```
Comment (kazuki43zoo) [inline source_en/ImplementationAtEachLayer/DomainLayer.rst]: - [ ] Why this change is need ? I think this change is not need.

**Comment by btratodos** [comment]
Closing this pull request as there are decrepencies in original source file and translated file.
